.env.example

```diff
@@ -0,0 +1 @@
+SITE_URL=https://lukeberrypi.vercel.app/
```

Comment:
might be good to use VERCEL_PROJECT_PRODUCTION_URL from here https://vercel.com/docs/projects/environment-variables/system-environment-variables#system-environment-variables